test_scripts/Smoke/Resumption/ATF_Resumption_big_amount_of_data.lua

```diff
@@ -0,0 +1,186 @@
+  end
+    local on_hmi_call = EXPECT_HMICALL("VR.AddCommand"):Times(20)
+    on_hmi_call:Do(function(_,data)
+      grammar_ID= data.params.grammarID
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+    EXPECT_RESPONSE("CreateInteractionChoiceSet", { success = true, resultCode = "SUCCESS" }):Times(20)
+    EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+      self.currentHashID = data.payload.hashID
+    end):Times(20)
+end
+
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Transport unexpected disconnect. Media app resume at FULL level")
+
+function Test:Close_Session()
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", {unexpectedDisconnect = true, 
+    appID = self.applications[default_app_params]})
+  self.mobileSession:Stop()
+end
+
+function Test:Register_And_Resume_App_And_Resumes_Data()
+  local mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  local on_rpc_service_started = mobileSession:StartRPC()
+  on_rpc_service_started:Do(function()  
+    config.application1.registerAppInterfaceParams.hashID = self.currentHashID
+    Test:expect_Resumption_Data()
+    commonStepsResumption:RegisterApp(self, default_app_params)   
+    commonStepsResumption:ExpectResumeAppFULL(self, default_app_params)
+  end)
+end
+
+function Test:expect_Resumption_Data()
+  on_ui_sub_menu_added = EXPECT_HMICALL("UI.AddSubMenu"):Times(20)
+  on_ui_sub_menu_added:Do(function(_,data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS")
+  end)
+  on_ui_sub_menu_added:ValidIf(function(_,data)
+      if data.params.appID == default_app_params.hmi_app_id then
+        return true
+      else 
+        commonFunctions:userPrint(31, "Media app is registered with wrong appID " )
+        return false
+      end
+  end)  
+  on_vr_commands_added = EXPECT_HMICALL("VR.AddCommand"):Times(40)
+  on_vr_commands_added:Do(function(_,data)
+    self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS")
+  end)
+  on_vr_commands_added:ValidIf(function(_,data)
+    if data.params.type == "Command" then 
+      if data.params.appID == default_app_params.hmi_app_id then
+        return true
+      else 
+        commonFunctions:userPrint(31, "Media app is registered with wrong appID ")
+        return false
+      end
+    elseif data.params.type == "Choice" then 
+      if data.params.appID == default_app_params.hmi_app_id then
+        return true
+      else 
+        commonFunctions:userPrint(31, "Media app is registered with wrong appID ")
+        return false
+      end
+    end
+  end)
+   self.mobileSession:ExpectNotification("OnHashChange")
+end
+
+function Test:OnCommand()
+  self.hmiConnection:SendNotification("UI.OnCommand",{ cmdID = 20, appID = default_app})
+  EXPECT_NOTIFICATION("OnCommand", {cmdID = 20, triggerSource= "MENU"})
+end
+
+function Test:PerformInteraction()
+  local cid = self.mobileSession:SendRPC("PerformInteraction",{ 
```

Comment:
@LitvinenkoIra I think you should add expectation of response on RPC "PerformInteraction" with current `cid` or remove it as unused variable.